change/@fluentui-web-components-a74d6518-0d94-4e22-bd9f-70ea63a066ab.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "adds switch component",
+  "packageName": "@fluentui/web-components",
+  "email": "email not defined",
```

Comment:
Please provide a valid email.
Configure your git client with the email so that `yarn change` fills it automatically next time